src/constants/tokenColors.ts

```diff
@@ -0,0 +1,73 @@
+export const DEFAULT_COLOR = [35, 43, 43]
+
+export const predefinedTokenColors: { [key: string]: number[] } = {
```

Comment:
I think this would be nicer if you find a way to use the address as opposed to the url.

Reply:
I looked into that for a little but you get into an issue with xchain tokens that have different address on different chains. They are currently covered by the current solution as they can have the same url for the image such as the coingecko url regardless of what chain they're on